pkg/platform/platform.go

```diff
@@ -87,9 +87,10 @@ type PlatformWithPostApplyHook interface { //nolint:golint
 
 // Meta is a generic information format about the platform.
 type Meta struct {
-	AssetDir      string
-	ExpectedNodes int
-	Managed       bool
+	AssetDir           string
+	ExpectedNodes      int
+	Managed            bool
+	ControlplaneCharts []helm.LokomotiveChart
```

Comment:
Not super keen on adding more fields to this kitchen sink but I guess it's fine for now. I hope we can do a platform refactor soon...

Reply:
True, but on the other hand I don't see much wrong in it. It's a static (or ideally it should be static) "manifest" of platform-specific information, it does not perform any action so we don't create some kind of multiplexing here.

Also this allows to easily add new optional fields for features for all platforms.